### PR TITLE
More release notes for 1.3.1

### PR DESCRIPTION
**Pull request by mheon**
Another PR slipped in

### CODE REVIEW

**Comment by mheon** [comment]
@TomSweeneyRedHat @vrothberg @giuseppe @baude PTAL

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 ### Features
 - The `podman cp` command can now read input redirected to `STDIN`, and output to `STDOUT` instead of a file, using `-` instead of an argument.
 - The Podman remote client now displays version information from both the client and server in `podman version`
+- The `podman unshare` command has been adding, allowing easy entry into the user namespace set up by rootless Podman (allowing the removal of files created by rootless Podman, among other things)
```
Comment (mtrmac) [inline RELEASE_NOTES.md]: `s/adding/added/`?

Comment (mheon) [reply RELEASE_NOTES.md]: Oops. Fixing.

**Comment by TomSweeneyRedHat** [comment]
Teach me to review before the first cup of tea.  Nice spot @mtrmac .
LGTM

**Pull request merged**